pkg/gui/context.go

```diff
@@ -231,12 +233,11 @@ func (gui *Gui) activateContext(c types.Context, opts ...types.OnFocusOpts) erro
 }
 
 func (gui *Gui) optionsMapToString(optionsMap map[string]string) string {
-	optionsArray := make([]string, 0)
```

Comment:
I must have written this back when I still carried my ruby intuitions about arrays vs slices